subsys/net/lib/sockets/sockets_tls.c

```diff
@@ -1184,6 +1197,66 @@ static int tls_opt_alpn_list_set(struct tls_context *context,
 	return 0;
 }
 
+#if defined(CONFIG_NET_SOCKETS_ENABLE_DTLS)
+static int tls_opt_dtls_handshake_timeout_get(struct tls_context *context,
+		void *optval, socklen_t *optlen, bool is_max)
```

Comment:
nit: Please try to align the arguments to the opening bracket to align with the rest of the code.

Reply:
Fixed all comments. I have a faint memory I read in Zephyr coding style that a tab is supposed to be 4 columns, but now I can't seem to find it. So FYI I am using 8 like the Linux coding style.